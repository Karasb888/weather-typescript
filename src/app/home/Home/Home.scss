.home {
    display: flex;
    justify-content: center;
}

.sun {
    width: 250px;
    height: auto;
}