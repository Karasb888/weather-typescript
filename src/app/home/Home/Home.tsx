import * as React from 'react';
import { Link } from 'react-router-dom';

const Home: React.FC = () => {
    return (
        <div>Home page stuf <Link to="/city/London/">Go to city</Link><Link to="/city/London/2020-2-17/">Go to Date</Link></div>
    );
};

export default Home;
