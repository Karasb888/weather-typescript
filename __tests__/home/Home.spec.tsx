import * as React from 'react';
import { shallow } from 'enzyme';
import Home from '../../src/app/home/Home/Home';

describe('++++ <Home /> ++++', () => {
    it('Renders correctly', () => {
        const wrapper = shallow(<Home />);
        expect(wrapper).toMatchSnapshot();
    });
});
