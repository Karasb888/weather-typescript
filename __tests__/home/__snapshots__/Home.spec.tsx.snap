// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`++++ <Home /> ++++ Renders correctly 1`] = `
<div
  className="home"
>
  <img
    alt="Fun sun gif animation"
    className="sun"
    src=""
  />
</div>
`;
